product-service/functions/addProduct.ts

```diff
@@ -21,22 +21,28 @@ const getDbOptions = () => {
   };
 }
 
-const addProductToDb = async ({ title, description, price }) => {
+const createQueryToAddProduct = async ({ title, description, price }) => {
+  if (!title || !description || !price) {
+      throw new Error('Insufficient fields');
+  }
+
+  const preparedProduct = `('${title}', '${description}', ${price})`;
+
+  const query = {
+      text: `insert into products(title, description, price) values ${preparedProduct}`,
+  }
```

Comment:
```
  const query = {
      text: `insert into products(title, description, price) values ($1, $2, $3)`,
      values: [title, description, price]
  }
```

It's better to use values to prevent sql injections.